bamboo/unit_tests/test_unit_checkpoint.py

```diff
@@ -37,7 +37,7 @@ def skeleton_checkpoint_lenet_shared(cluster, executables, dir_name,
         cluster=cluster, executable=exe, num_nodes=1, num_processes=2,
         dir_name=dir_name,
         data_filedir_default='/p/lscratchh/brainusr/datasets/MNIST',
-        data_reader_name='mnist', data_reader_percent=data_reader_percent,
+        data_reader_name='mnist', data_reader_percent=0.01,
```

Comment:
Since this is repeated multiple times, I would store it in a variable, so if a change is necessary then we only have to do it in one place.

Reply:
Fixed.